### DISCUSSION TITLE
I want "show" a FluentPanel without displaying it

### DISCUSSION THREAD

**Discussion by SpenceDahl** [microsoft/fluentui-blazor#2795] [Q&A] [answered]
I want to use the Dialog Service’s ShowPanelAsync to “open” a `<FluentPanel />` so its code can run, but the panel doesn’t initially display. I’ve tried setting Visible to false, and as expected, the panel does not display, but the code doesn’t run either. Is this possible?

**Comment by vnbaaij** [comment, marked as answer]
No, this is not possible currently (and I don't think this is donating we will be adding anytime soon).
